### PR TITLE
C3VOC tests adjustments

### PR DESCRIPTION
**Pull request by jayaddison**
Miscellaneous changes - note that behaviour adjustments (and token value changes) are included, so this isn't strictly a refactoring.

### CODE REVIEW

File: tests/test_api_c3voc.py [hunk 1]
```diff
@@ -179,24 +140,19 @@ def test_update_voctoweb_with_correct_url(client, app, db, proposal):
 
     proposal = Proposal.query.get(proposal.id)
     assert proposal.c3voc_url == "https://media.ccc.de/"
-    assert proposal.video_recording_lost == False
+    assert proposal.video_recording_lost is False
```
Comment (fkusei) [inline tests/test_api_c3voc.py:143]: Setup should set this to `True` to make sure the webhook sets it to `False`.

File: tests/test_api_c3voc.py [hunk 2]
```diff
@@ -440,24 +348,19 @@ def test_update_youtube_with_correct_url(client, app, db, proposal):
 
     proposal = Proposal.query.get(proposal.id)
     assert proposal.c3voc_url is None
-    assert proposal.video_recording_lost == False
+    assert proposal.video_recording_lost is False
```
Comment (fkusei) [inline tests/test_api_c3voc.py:351]: Setup should set this to `True` to make sure the webhook sets it to `False`.

**Review by fkusei — CHANGES_REQUESTED** [review]
`clean_proposal()` set `video_recording_lost` to True by default.

**Review by Kunsi — APPROVED** [review]

**Comment by jayaddison** [comment]
Thanks for the code review @Kunsi!  Merging this in a few moments..

**Review by russss — APPROVED** [review]

**Comment by jayaddison** [comment]
Thanks @russss!

**Pull request merged**